src/modules/site-v2/base/views/tools/pairwise_indel_finder.py

```diff
@@ -304,29 +303,26 @@ def report(id, file_ext=None):
       logger.error(f'Invalid interval "{data["site"]}" for Indel Finder report {id}')
       indel_start, indel_stop = None, None
 
+    # Extract the dataframe from the results
+    dataframe = result.get('dataframe', None)
 
-    # Update the result object with computed fields and generate a format table
-    # If result is None or empty, does nothing
-    result, format_table = modify_indel_primer_result(result)
```

Comment:
Instead of the somewhat ad-hoc `modify_indel_primer_result` function, this processing is now part of the `IndelPrimerJobPipeline` class's fetching logic